src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
```

Comment:
good practice to decompose `action` object
`{ extraData, keysToRemove, ... } = action;`